static/test/javascripts-legacy/spec/facia/weather.spec.js

```diff
@@ -16,7 +16,7 @@ define([
         beforeEach(function (done) {
 
             injector.require(['facia/modules/onwards/weather', 'lib/config'], function () {
-                sut = arguments[0];
+                sut = arguments[0].Weather;
```

Comment:
I did try moving this over to Jest/ES6 etc., but that seems complicated until `lib/detect.js` is ported over.